src/Uno.UI/UI/Xaml/Shapes/ArbitraryShapeBase.wasm.cs

```diff
@@ -23,6 +24,13 @@ private IDisposable BuildDrawableLayer()
 
 		protected override Size MeasureOverride(Size availableSize)
 		{
+			// We make sure to invoke native methods while not in the visual tree
+			// (For instance getBBox will fail on FF)
+			if (Parent == null)
+			{
+				return new Size();
```

Comment:
```suggestion
				return default;
```

Reply:
Not supported properly by build servers yet ... already spent way to much time to undo this kind of changes!